addons/sale/migrations/11.0.1.1/post-migration.py

```diff
@@ -8,9 +9,20 @@
 ]
 
 
+def activate_proforma(env):
+    """If module `sale_proforma_report` (OCA/sale-reporting) was installed we
+    need to activate the "Pro-forma Invoices" setting as the module was
+    deprecated in favour of it."""
+    if openupgrade.column_exists(env.cr, 'sale_order', 'proforma'):
+        employee_group = env.ref('base.module_category_human_resources')
+        proforma_group = env.ref('sale.group_proforma_sales')
+        employee_group.write({'implied_ids': [(4, proforma_group.id)]})
```

Comment:
Is this the way that Odoo uses when you mark the field in settings?

Reply:
yes, exactly, I took it from res.settings.

Reply:
btw, I already tested it and worked :green_heart: